node-dogstatsd/node-dogstatsd.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for Datadog's nodejs metrics client node-dogstatsd
+// Project: https://github.com/datadog/node-dogstatsd
```

Comment:
@vvakame Good catch, the url has been corrected. This is the repo that Datadog's official integration instructions point to.